sbtc/src/events.rs

```diff
@@ -11,24 +11,33 @@
 use std::collections::BTreeMap;
 
 use bitcoin::hashes::Hash;
-use bitcoin::BlockHash as BitcoinBlockHash;
 use bitcoin::OutPoint;
 use bitcoin::PubkeyHash;
 use bitcoin::ScriptBuf;
 use bitcoin::ScriptHash;
 use bitcoin::Txid as BitcoinTxid;
 use bitcoin::WitnessProgram;
 use bitcoin::WitnessVersion;
-use bitvec::array::BitArray;
-use blockstack_lib::burnchains::Txid as StacksTxid;
 use clarity::vm::types::CharType;
 use clarity::vm::types::PrincipalData;
 use clarity::vm::types::SequenceData;
 use clarity::vm::types::TupleData;
 use clarity::vm::ClarityName;
 use clarity::vm::Value as ClarityValue;
 use secp256k1::PublicKey;
-use stacks_common::types::chainstate::StacksBlockId;
+use stacks_common::*;
+use stacks_common::types::chainstate::{BurnchainHeaderHash, StacksBlockId};
+
+/// Stacks transaction identifier. Wrapper over a 32 byte array.
+pub struct StacksTxid(pub [u8; 32]);
+// impl_byte_array_newtype!(StacksTxid, u8, 32);
+
+
+impl_array_newtype!(StacksTxid, u8, 32);
+impl_array_hexstring_fmt!(StacksTxid);
+impl_byte_array_newtype!(StacksTxid, u8, 32);
+impl_byte_array_message_codec!(StacksTxid, 32);
+impl_byte_array_serde!(StacksTxid);
```

Comment:
Do we need all of these? I'm under the impression that we don't, and that we can just have the `StacksTxid(pub [u8; 32])` type. Maybe just derive `Copy, Clone, Debug`.